lib/models/listwidget.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:flutter/material.dart';
 
-abstract class ListItem {
```

Comment:
I removed this abstract class for now. All my sections implemented this class.

But many of section widgets dont need 'head' and it can be handled inside each section class which simply extends StatelessWidget.

I think it was not fully well used for each section.

I think that the important point was complex widget can be handled separately. So I keep the structure to do that.

Do I get it right?